src/indexing.c

```diff
@@ -17,6 +17,8 @@
 #include "mufs.h"
 #include "db.h"
 
+static char *untagged = "<Untagged>";
```

Comment:
<Untagged> may interfere with file paths and looks a bit unpleasant, just "Untagged" would be better

Reply:
Sure.